### PR TITLE
add Scaleway catalog

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

**Review by concretevitamin — APPROVED** [review]
LGTM. Tagging @Michaelvll @romilbhardwaj to take a look too.

File: catalogs/v5/scaleway/vms.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+AvailabilityZone,InstanceType,vCPUs,MemoryGiB,AcceleratorName,AcceleratorCount,GPUInfo,Region,Price,SpotPrice
+fr-par-1,RENDER-S,10,45097156608,P100,1,"{'Gpus': [{'Name': 'Tesla P100-PCIE-16GB', 'Manufacturer': 'NVIDIA', 'Count': 1, 'MemoryInfo': {'SizeInMiB': 16384}}], 'TotalGpuMemoryInMiB': 16384}",fr-par,1.2426,
```
Comment (Michaelvll) [inline catalogs/v5/scaleway/vms.csv:2]: The Memory column should using the unit of GiB. Please double check the memory number here.

**Review by Michaelvll — COMMENTED** [review]
Thanks for adding this @remyleone! The memory size may need to be fixed. Otherwise, it looks good to me.

File: catalogs/v5/scaleway/vms.csv (hunk 1 above)
Comment (remyleone) [reply catalogs/v5/scaleway/vms.csv]: Fixed :)

**Review by Michaelvll — APPROVED** [review]
Thanks for the quick fix @remyleone! LGTM.

**Pull request merged**